coverage/sqldata.py

```diff
@@ -774,7 +774,7 @@ def erase(self, parallel=False):
         file_be_gone(self._filename)
         if parallel:
             data_dir, local = os.path.split(self._filename)
-            localdot = local + ".*"
+            localdot = glob.escape(local) + ".*"
             pattern = os.path.join(os.path.abspath(data_dir), localdot)
```

Comment:
The `data_dir` should also be escaped here, there is no guarantee that it won't have `[` or `]` in it either